docs/source/export.rst

```diff
@@ -291,6 +291,176 @@ torch.export>`), but seeing as the context manager does not affect the tensor
 computations in the model, we can go with the non-strict mode's result.
 
 
+.. _Training Export:
+
+Training Export
+^^^^^^^^^^^^^^^^^
+
+In PyTorch 2.5, we introduced a new API called :func:`export_for_training`.
+It's still going through hardening, so if you run into any issues, please file
+them to Github with the "oncall: export" tag.
+
+In this API, we produce the most generic IR that contains all ATen operators
+(including both functional and non-functional) which can be used to train in
+eager PyTorch Autograd. This API is intended for PT2 quantization training use cases
```

Comment:
I'm not sure why you need to mention quantization training. E.g., you'd want this for normal eager training too.